rlmeta/core/remote.py

```diff
@@ -31,10 +31,20 @@ def __new__(cls, name, bases, attrs):
 
 class Remotable(abc.ABC, metaclass=RemotableMeta):
 
+    def __init__(self, identifier: str = ""):
```

Comment:
RemotableModel is also inherited from Remotable and actually that's the main reason we want this feature.
https://github.com/facebookresearch/rlmeta/blob/main/rlmeta/core/model.py#L17